js/src/components/google-account-card/connect-google-account-card.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import AccountCard, { APPEARANCE } from '.~/components/account-card';
+import AppButton from '.~/components/app-button';
+import readMoreLink from './read-more-link';
+import useGoogleConnectFlow from './use-google-connect-flow';
+
+export default function ConnectGoogleAccountCard( { disabled } ) {
+	const [ handleConnect, { loading, data } ] = useGoogleConnectFlow();
+
+	return (
+		<AccountCard
+			appearance={ APPEARANCE.GOOGLE }
+			disabled={ disabled }
+			alignIcon="top"
+			description={
+				<>
+					{ __(
+						'Required to sync with Google Merchant Center and Google Ads.',
+						'google-listings-and-ads'
+					) }
+					<p>
+						<em>
+							{ createInterpolateElement(
+								__(
+									'You will be prompted to give WooCommerce access to your Google account. Please check all the checkboxes to give WooCommerce all required permissions. <link>Read more</link>',
+									'google-listings-and-ads'
+								),
+								{
+									link: readMoreLink,
+								}
+							) }
+						</em>
+					</p>
+				</>
+			}
+			alignIndicator="top"
+			indicator={
+				<AppButton
+					isSecondary
+					disabled={ disabled }
+					loading={ loading || data }
+					eventName="gla_google_account_connect_button_click"
```

Comment:
💅 
Maybe add [the `eventProps`](https://github.com/woocommerce/google-listings-and-ads/pull/1069/commits/c4761c141286e19c937f41e89036871021ad30d6#diff-985cbb1246d5a14d2eb12acaf614585f17c15b4d6062c1c224c959a6a781387dR119-R122) first to avoid missing it when dealing with merging conflicts, or missing it directly if no conflicts happened.

Reply:
Done in [`65478e6` (#1074)](https://github.com/woocommerce/google-listings-and-ads/pull/1074/commits/65478e6823aa646e98ac46902759d085100671a4).